examples/common_patterns_for_connectors/priority_first_sync_for_high_volume_initial_syncs/mock_api.py

```diff
@@ -0,0 +1,38 @@
+
+from fivetran_connector_sdk import Logging as log
+from faker import Faker
+from datetime import datetime, timedelta, timezone
+
+import connector
+
+fake = Faker()
+
+def get_api_response(base_url, params):
+    log.info(f"Making API call to url: {base_url} with params: {params}")
```

Comment:
We can have an endpoint for this mock data generator in our playground. Not as part of this ticket, but you can do it later.
As I am using a free API for the user in `user_profiles` examplese, I will be able to reuse the same endpoint there as well.
cc: @varundhall

Reply:
Yes will create a follow up ticket.